backend/app/schemas/quiz_check.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import BaseModel
+from typing import List, Any
+
+
+class Smell(BaseModel):
+    """A class to represent a code smell in a file"""
+
+    start: int
+    end: int
+    type: str
+
+
+class File(BaseModel):
+    """A class to represent a file and its code smells"""
+
+    file_url: str
```

Comment:
Pydantic has a type for validating a URL

https://docs.pydantic.dev/latest/api/networks/

Reply:
Shouldn't we then change Quiz.file_urls type?